test/e2e/network/netpol/kubemanager.go

```diff
@@ -70,10 +69,14 @@ func (k *kubeManager) initializeCluster(model *Model) error {
 			}
 
 			createdPods = append(createdPods, kubePod)
-			_, err = k.createService(pod.Service())
+			svc, err := k.createService(pod.Service())
 			if err != nil {
 				return err
 			}
+			if net.ParseIP(svc.Spec.ClusterIP) == nil {
+				return fmt.Errorf("empty IP address found for service %s/%s", svc.Namespace, svc.Name)
```

Comment:
can this only be an empty IP at this point?

if not, might be better to reword:
```
return fmt.Errorf("invalid IP address found for service %s/%s: %s", svc.Namespace, svc.Name, svc.Spec.ClusterIP)
```

https://pkg.go.dev/net#ParseIP
> If s is not a valid textual representation of an IP address, ParseIP returns nil.